src/main/java/Duke.java

```diff
@@ -28,9 +29,26 @@ public static void main(String[] args) {
                 System.out.println("Nice! I've marked this task as done:");
                 System.out.printf("  [%s] %s\n",tasks[FinishedNumber-1].getStatusIcon(),tasks[FinishedNumber-1].getDescription());
             } else {
+                System.out.println("Got it. I've added this task:");
+                String[] splitString = next.split(" ", 2);
```

Comment:
I think you should use more descriptive variable names than splitString, splitString2.
Maybe you can replace splitString with something like firstWord or command